libcusp/include/galois/graphs/DistributedGraph.h

```diff
@@ -1006,6 +994,18 @@ class DistGraph {
     galois::gDebug("Deallocating CSR in DistGraph");
     graph.deallocate();
   }
+
+  /**
+   * Sort the underlying LC_CSR_Graph by ID (destinations)
+   */
+  void sortEdges() {
```

Comment:
More descriptive name please - sortEdgesByDestination() is fine but technically it is sortEdgesOfNodesByDestination